assets/app/scripts/controllers/overview.js

```diff
@@ -289,26 +291,26 @@ angular.module('openshiftConsole')
         return;
       }
       // Make sure we have a build output
-      if (!build.parameters.output.to) {
+      if (!build.spec.output.to) {
         return;
       }
-      for (var i = 0; i < deploymentConfig.triggers.length; i++) {
-        var trigger = deploymentConfig.triggers[i];
+      for (var i = 0; i < deploymentConfig.spec.triggers.length; i++) {
+        var trigger = deploymentConfig.spec.triggers[i];
         if (trigger.type === "ImageChange") {
           var triggerImage = trigger.imageChangeParams.from.name;
```

Comment:
@ironcladlou if a deployment's ImageChange trigger has `imageChangeParams.from=={kind:"ImageStream",name:"foo"}`, is that same as triggering off the ImageStreamTag foo:latest?